pages/search.js

```diff
@@ -0,0 +1,225 @@
+import React, { useCallback, useState, useEffect } from "react";
+import Header from "../components/Shared/Header";
+import Footer from "../components/Shared/Footer";
+import ScrollToTop from "../components/Shared/ScrollToTop";
+
+import { themeSetting } from "../components/Shared/UISettings";
+import dynamic from 'next/dynamic';
+
+const BarcodeScanner = dynamic(
```

Comment:
This is only useful for the intro page and we do not need it here.

Reply:
Yes alright understood, I was trying to copy the intro page functionalities so thought it was necessary .